cloud/api/protobuf/kurtosis_backend_server_api.proto

```diff
@@ -281,4 +283,17 @@ message  AddAliasRequest {
   string access_token=1;
   Port port=2;
   string alias=3;
-}
\ No newline at end of file
+}
+
+message IsNewKurtosisVersionAvailableRequest {
+  string current_kurtosis_version = 2;
```

Comment:
Those requests go through the cloud gateway afaik so it should contain an access token.

Reply:
@laurentluce access token added